examples/crypto-websockets/server/asyncapi.yaml

```diff
@@ -8,6 +8,12 @@ servers:
   websocket:
     url: ws://localhost:3000
     protocol: ws
+    security:
+      - tokens: []
+      - JWT: []
```

Comment:
I might be wrong but shouldn't you reference the `securitySchemes` object.

Reply:
No, that's not needed. The spec file matches them using the name. For each `security`, it checks if there's a matching `securityScheme`.